src/extractors/mfcc.js

```diff
@@ -31,14 +31,12 @@ export default function(args) {
     }
 
     //log each coefficient unless it's 0.
```

Comment:
Oops, no it's not. I have it removed locally and I'll push it once I fix the tests.